taskcluster/firefox_tv_taskgraph/email.py

```diff
@@ -18,23 +18,20 @@
 @transforms.add
 def email_task(config, tasks):
     tag = config.params.get("head_tag")
+    release_type = config.params.get("release_type")
 
     for task in tasks:
         to_address = task["worker"].pop("to-address")
-        content = task["worker"].pop("content")
+        content_chooser = task["worker"].pop("content")
+        content = content_chooser["by-release-type"].get(release_type, "")
```

Comment:
Nit: Use `resolve_keyed_by()`. It handles all cases for you. For instance: https://github.com/mozilla-mobile/fenix/blob/6a8f7224dc6c583f480a258ba7af796fa562e935/taskcluster/fenix_taskgraph/transforms/signing.py#L22